upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -1025,15 +1025,19 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 
 	if b.cluster.Spec.Networking.AmazonVPC != nil {
 		key := "networking.amazon-vpc-routed-eni"
-		version := "1.5.0-kops.1"
-
+		versions := map[string]string{
+			"k8s-1.7":  "1.3.3-kops.1",
+			"k8s-1.8":  "1.3.3-kops.1",
+			"k8s-1.10": "1.3.3-kops.1",
```

Comment:
The problem with moving the version down, is that I think is that this means that we won't apply updates to existing kops cluster if the new version is < 1.5.0-kops.1.  It doesn't matter here where we aren't changing the manifest, but if there was a security update to 1.3.4 and we bumped to 1.3.4-kops.1, I think that would not be applied.

I know that in practice the new version would probably be 1.5.4 and this wouldn't happen.    But can we leave it with the old versions as-is (maybe with a note saying "this version number is misleading")?

Reply:
makes sense, I've updated the PR to keep older k8s versions on the original 1.5.0-kops.1 manifest version